src/main/kotlin/ro/hibyte/betting/dto/EventRequest.kt

```diff
@@ -0,0 +1,17 @@
+package ro.hibyte.betting.dto
+
+import jakarta.persistence.ElementCollection
+import ro.hibyte.betting.entity.Status
+import java.sql.Timestamp
+import java.time.Instant
+
+data class EventRequest(
```

Comment:
What is an `EventRequest`? Is it a DTO or is it something that will be persisted in the database?

Reply:
it is a DTO